### PR TITLE
fix unknown kraken symbol raised error

### PR DESCRIPTION
**Pull request by iamzoltan**
changed the error raised when the symbol passed to kraken does not match any of their tickers.

### CODE REVIEW

File: piker/brokers/kraken.py [hunk 1]
```diff
@@ -174,7 +174,8 @@ async def symbol_info(
         resp = await self._public('AssetPairs', pairs)
         err = resp['error']
         if err:
-            raise BrokerError(err)
+            symbolname = pairs['pair'] if pair else None
+            raise SymbolNotFound(f'kraken: {symbolname}')
```
[the remark this answers is not in the document]
Comment (iamzoltan) [reply piker/brokers/kraken.py:178]: im into that idea!

[the remark this answers is not in the document]
Comment (goodboy) [reply piker/brokers/kraken.py:178]: I think that way we kind of unify the naming across providers?

[the remark this answers is not in the document]
Comment (iamzoltan) [reply piker/brokers/kraken.py]: just pushed the change

**Review by goodboy — APPROVED** [review]
Slick. @guilledk any quiffs with this?

**Comment by guilledk** [comment]
Nope, looks great to me

**Pull request merged**